source/form.html

```diff
@@ -0,0 +1,198 @@
+              </p>
+            </li>
+            <li class="objective__item">
+              <p class="objective__slim">
+                <input class="objective__slim-input" type="radio" id="advice" name="type-objective">
+                <label class="objective__slim-text" for="advice">Не знаю (Нужен ваш совет)</label>
+              </p>
+            </li>
+          </ul>
+        </fieldset>
+        <fieldset class="information">
+          <legend class="information__title">Контактные данные
+            (владельца кота)</legend>
+          <ul class="information__list">
+            <li class="information__item">
+              <p class="information__email">
+                <label class="information__email-name" for="mail">E-mail:*</label>
+                <input class="information__email-input" type="text" id="mail" placeholder="kuklachev@gmail.com">
+              </p>
+            </li>
+            <li class="information__item">
+              <p class="information__phone">
+                <label class="information__phone-name" for="phone">Телефон:*</label>
+                <input class="information__phone-input" type="text" id="phone" placeholder="8 (960) 900-60-90">
+              </p>
+            </li>
+          </ul>
+        </fieldset>
+        <fieldset class="comments">
+          <legend class="comments__title">Комментарий</legend>
+          <p class="comments__item">
+            <input class="comments__input" type="text" placeholder="Расскажите обо всех повадках кота">
+          </p>
+        </fieldset>
+        <fieldset class="optionally">
+          <legend class="optionally__title">Дополнительно</legend>
+          <ul class="optionally__list">
+            <li class="optionally__item">
+              <p class="optionally__unit">
+                <input class="optionally__input" type="checkbox" id="sweet">
+                <label class="optionally__title-label" for="sweet">сахарозаменитель</label>
+              </p>
+            </li>
+            <li class="optionally__item">
+              <p class="optionally__unit">
+                <input class="optionally__input" type="checkbox" id="water">
+                <label class="optionally__title-label" for="water">питьевая вода</label>
+              </p>
+            </li>
+            <li class="optionally__item">
+              <p class="optionally__init">
+                <input class="optionally__input" type="checkbox" id="milk">
+                <label class="optionally__title-label" for="milk">молоко</label>
+              </p>
+            </li>
+            <li class="optionally__item">
+              <p class="optionally__unit">
+                <input class="optionally__input" type="checkbox" id="vitamin">
+                <label class="optionally__title-label" for="vitamin">витамины</label>
+              </p>
+            </li>
+          </ul>
+          <button class="optionally__button" type="button">отправить заявку</button>
```

Comment:
```suggestion
          <button class="optionally__button" type="submit">отправить заявку</button>
```
так как кнопка с типом `button` не отправит данные из формы.